package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aol-hb",
-  "version": "0.9.4",
```

Comment:
Because it is a change in the library and we have to publish as a newer version correct?